src/pages/BooksOverview/BooksOverview.tsx

```diff
@@ -0,0 +1,17 @@
+import BooksList from 'components/BooksList/BooksList';
+import Navbar from 'components/Navbar/Navbar';
+import React from 'react';
+import classes from './BooksOverview.module.scss';
+
+const BooksOverview = () => {
+  return (
+    <div>
+      <Navbar />
```

Comment:
We don't want to import `<Navbar />` component in all pages separately, instead you can create Layout component and add `<Navbar />` there, or you can put it in App above router import